etc/generate_tiles.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+from math import pi,cos,sin,log,exp,atan
+from subprocess import call
+import sys, os
+from Queue import Queue
+
+import threading
+
+try:
+    import mapnik2 as mapnik
+except:
+    import mapnik
+
+DEG_TO_RAD = pi/180
+RAD_TO_DEG = 180/pi
+
+# Default number of rendering threads to spawn, should be roughly equal to number of CPU cores available
+NUM_THREADS = 4
+
+
+def minmax (a,b,c):
+    a = max(a,b)
+    a = min(a,c)
+    return a
+
+class GoogleProjection:
```

Comment:
Can we add some documentation to this? it's not really clear what any of the variables are here.

Reply:
tbh I haven't looked into how this script works, I've only altered the configuration (last two lines); it's the standard Mapnik script from https://github.com/openstreetmap/mapnik-stylesheets/blob/master/generate_tiles.py

Sorry, I should have mentioned it somehow.

Reply:
Can you add a comment to the top of the file?

Reply:
Done